src/test/java/teammates/test/cases/browsertests/InstructorFeedbackResultsPageResponseCommentUiTest.java

```diff
@@ -13,6 +11,7 @@
  *      specifically for feedback response comments.
  */
 public class InstructorFeedbackResultsPageResponseCommentUiTest extends BaseUiTestCase {
+    private InstructorFeedbackResultsPage resultsPage;
```

Comment:
Don't put the `resultsPage` as a attribute. `@Test` might be run concurrently and result in unexpected behavior.